### PR TITLE
fix(docs) : Fix navbar items UI

### LINKED ISSUE #4129: Fix the navbar highlighting issue
the navbar in our docs has a bug, whenever the TS SDK tab is clicked, the References tab also gets highlighted.

Image

### PR DESCRIPTION
**Pull request by vivekjain23**
# Description of change

This PR bring up the changes to fix the navbar items UI. It removes TS SDK from the references directory. 

## Links to any relevant issues

fixes #4129.

## Type of change

- Documentation Fix

## How the change has been tested

Changes were built locally.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@vivekjain23 , what's the status on this PR?

**Comment by vivekjain23** [comment]
> @vivekjain23 , what's the status on this PR?

Had a blocker, regarding the `client-redirect` plugin, resolved it and now it is ready to review.

File: docs/site/docusaurus.config.js [hunk 1]
```diff
@@ -108,6 +108,214 @@ const config = {
         intentionallyNotExported: [],
       },
     ],
+    [
```
Comment (Dr-Electron) [inline docs/site/docusaurus.config.js:113]: This should be done with a wildcard, no?

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: docs/site/docusaurus.config.js [hunk 2]
```diff
@@ -108,6 +108,214 @@ const config = {
...
+          },
+          {
+              to: 'ts-sdk/kiosk/advanced-examples',
+              from: '/references/ts-sdk/kiosk/advanced-examples',
+          },
+          {
+              to: 'ts-sdk/bcs',
+              from: '/references/ts-sdk/bcs',
+          },
+          ];
```
Comment (Dr-Electron) [inline docs/site/docusaurus.config.js]:
```suggestion
            {
              from: '/references/ts-sdk',
              to: '/ts-sdk',
            },
          ];
```

File: docs/site/docusaurus.config.js (hunk 1 above)
Comment (vivekjain23) [reply docs/site/docusaurus.config.js:113]: Yes, why not? 
Will update in the next commit. 🙌

**Review by begonaalvarezd — APPROVED** [review]

**Review by lucas-tortora — APPROVED** [review]

**Review by Ginowine — APPROVED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request closed without merging**